news/7833.removal

```diff
@@ -0,0 +1 @@
+Remove duplicate ``tests.lib.path.Path.resolve`` method
```

Comment:
Since this is not user facing, you can drop the new file. I've marked the PR as trivial so it is not required anymore.